ChineseFilter/sections/general_extensions.txt

```diff
@@ -59,12 +59,13 @@ ucptt.com#%#(function() { window.open_ = window.open; var w_open = window.open,
 goodav17.com#%#//scriptlet("abort-on-property-read", "popunder")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/132934
 v-wb.youku.com#%#//scriptlet('set-constant', 'Object.prototype.adData', 'emptyObj')
+! https://github.com/AdguardTeam/AdguardFilters/issues/141905
 ! https://github.com/AdguardTeam/AdguardFilters/issues/121307
 ! https://github.com/AdguardTeam/AdguardFilters/issues/87176
 ! https://github.com/AdguardTeam/AdguardFilters/issues/54359
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26318
 ! https://forum.adguard.com/index.php?threads/youku-com-v-qq-com.30749/
-youku.com#%#//scriptlet("adjust-setTimeout", "r--", "", "0.02")
+youku.com#%#//scriptlet('adjust-setTimeout', 'e-', '', '0.02')
```

Comment:
Does player work on your end?

Reply:
Yup. With Mulklvad VPN, the player works well.

Reply:
I still need the old rule (`youku.com#%#//scriptlet("adjust-setTimeout", "r--", "", "0.02")`) otherwise I see ad timer.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/217521884-21b713d5-c052-4a6f-a758-2cac8589f0e8.png)

</details>

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/142229#discussion_r1100024697 I also need this rule, but don't see ad on my end.